seed/validate.go

```diff
@@ -78,3 +79,14 @@ func ValidateFromYaml(seedYamlFile string) error {
 
 	return nil
 }
+
+var validSeedSystemLabel = regexp.MustCompile("^[a-zA-Z0-9][a-zA-Z0-9-]+$")
```

Comment:
I suppose it's fine if the label ends with "-"?

Reply:
The code complains about `*-` now.